### PR TITLE
Main page

### PR DESCRIPTION
**Pull request by tlj4k8**
Started home page and created a navbar.

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -1,18 +1,163 @@
 <template>
   <div class="home">
-    <img src="../assets/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js App"/>
-  </div>
+      <NavBar></NavBar>
+      <br>
```
Comment (dborstelmann) [inline src/views/Home.vue]: Never use BRs unless you have a reason

File: src/components/NavBar.vue [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<template>
+    <el-menu
+        :default-active="Index2"
```
Comment (dborstelmann) [inline src/components/NavBar.vue]: Does this work now?  I know there was an error here yesterday

File: src/views/Home.vue [hunk 3]
```diff
@@ -1,18 +1,163 @@
...
 <script>
-// @ is an alias to /src
-import HelloWorld from "@/components/HelloWorld.vue";
+import NavBar from "@/components/NavBar.vue";
 
 export default {
   name: "home",
+  data() {
+    return {
+      tableData: [
```
Comment (dborstelmann) [inline src/views/Home.vue]: This is great.  We could even move this to the database for more modularity at a later date.

File: src/components/NavBar.vue (hunk 2 above)
Comment (tlj4k8) [reply src/components/NavBar.vue]: It works, but still shows and error.

File: src/views/Home.vue (hunk 3 above)
Comment (tlj4k8) [reply src/views/Home.vue]: Okay sounds good!

File: src/components/NavBar.vue (hunk 2 above)
Comment (dborstelmann) [reply src/components/NavBar.vue]: Ok we can fix it later

**Pull request merged**